app.js

```diff
@@ -131,20 +131,12 @@ app.post('/api/v1/palettes', async (request, response) => {
 
   for(let reqParam of ['palette_title', 'color_1', 'color_2', 'color_3', 'color_4', 'color_5', 'project_title']) 
     if(!newPalette[reqParam]) {
-      return response.status(422).json({error: `expected format {
-      palette_title: <String>,
-      color_1: <String>,
-      color_2: <String>,
-      color_3: <String>,
-      color_4: <String>,
-      color_5: <String>,
-      project_title: <String> 
-     } You are missing ${reqParam}`
+      return response.status(422).json({error: `expected format {palette_title: <String>, color_1: <String>, color_2: <String>, color_3: <String>, color_4: <String>, color_5: <String>, project_title: <String> } You are missing ${reqParam}`
```

Comment:
I  think the old way made it more readable

Reply:
Yes i was trying to get the test to pass and trying to format it the same way it was didn't work. I had teh same information structured the same way and test failed until i moved it back to a single line.

Reply:
ok thanks